ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.ts

```diff
@@ -12,6 +12,15 @@ export interface Data extends RaidbossData {
   seedTargets?: string[];
   seenSphere?: boolean;
   signalCount?: number;
+  deployArmaments?: number[];
+}
+
+// Enumeration of possible Deploy Armaments attacks
+enum DeployArmaments {
```

Comment:
Because users can look at transpiled triggers via the triggers branch, I think it'd be better to just use a JavaScript object here and not a TypeScript enum, if you don't mind.

Reply:
Do you have an example for this?